src/utils/filters.ts

```diff
@@ -0,0 +1,167 @@
+// @ts-ignore
+import { geojsonToWKT } from "@terraformer/wkt";
+import { Geometry } from "geojson";
+
+export {
+  and,
+  or,
+  intersects,
+  not,
+  equals,
+  lessThan,
+  lessThanOrEqual,
+  greaterThan,
+  greaterThanOrEqual,
+  like,
+  between,
+  isin,
```

Comment:
I'm torn about whether we should be abstracting the CQL filters. Although it made sense when the filters were xml and difficult to understand, CQL seems to be basically plaintext and might just be easier to interact with directly rather than with our methods. The only exception to that I could see is for intersects, because it's good to abstract away the WKT, so this method might be worth keeping just in case more geospatial filters get added.

Reply:
Will discuss in the scrum and I'll pause any refactoring etc. for the filters for the time being